webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+const SRC_DIR = path.join(__dirname, '/client/src');
+const DIST_DIR = path.join(__dirname, '/public');
+
+module.exports = {
+  entry: `${SRC_DIR}/index.jsx`,
```

Comment:
Maybe path.resolve here?

Reply:
Used path.join. Thanks for the heads-up!